RELEASING.md

```diff
@@ -238,6 +249,12 @@ as described in the section above.
    git add _data/release-notes/<VERSION>-release-notes.md
    ```
 
+1. In [netlify.toml](netlify.toml):
+      
+      1. Set the `RELEASE_VERSION` environment variable to new version.
+      
+1. On netlify update `Production branch` in Deploy contexts to release-vX.Y in docs.projectcalico.org site settings.
```

Comment:
does this need to be done before or after the surrounding changes are merged? If before, can you move it up to be step 1? And if after, can you move it down after step 10?

Having it sit in between is a bit odd because the user is in the middle of  building a PR to master

Reply:
I think that's doable. I added it in between because doing it before the commit will trigger the build. But we can ask the user to trigger the build manually after updating the production branch.